tests/Doctrine/Tests/ORM/Hydration/ObjectHydratorTest.php

```diff
@@ -1959,11 +1959,11 @@ public function testMissingDiscriminatorColumnException() : void
 
     /**
      * @group DDC-3076
-     * @expectedException \Doctrine\ORM\Internal\Hydration\HydrationException
-     * @expectedExceptionMessage The discriminator value "subworker" is invalid. It must be one of "person", "manager", "employee".
      */
     public function testInvalidDiscriminatorValueException() : void
     {
+        $this->expectException('Doctrine\ORM\Internal\Hydration\HydrationException');
```

Comment:
Like mentioned above, please move this before the affected code.
I'll go into lazy mode again. Please check the other exception tests for this too.